src/nomadic/realtime/dashboard/translations/en.yml

```diff
@@ -0,0 +1,29 @@
+en:
+  # Banner
+  Experiment Name: Experiment Name
+  Started at: Started at
+  Time elapsed: Time elapsed
+  FASTQs Processed: FASTQs Processed
+  # Mapping
+  n_primary: Primary
+  n_secondary: Secondary
+  n_chimeria: Chimera
+  n_unmapped: Unmapped
+  n_primary_tooltip: Number of primary alignments. Primary alignment is the best or main alignment of a read to the reference genome.
```

Comment:
These tooltip definitions are nice, good work. We should unify with the documentation.